sdk/storage/azure-storage-file/swagger/README.md

```diff
@@ -48,4 +48,51 @@ directive:
   transform: >
     $.Start.format = "str";
     $.Expiry.format = "str";
-```
\ No newline at end of file
+- from: swagger-document
+  where: $["x-ms-paths"]..responses..headers["x-ms-file-last-write-time"]
+  transform: >
+    $.format = "str";
+- from: swagger-document
+  where: $["x-ms-paths"]..responses..headers["x-ms-file-change-time"]
+  transform: >
+    $.format = "str";
+- from: swagger-document
+  where: $["x-ms-paths"]..responses..headers["x-ms-file-creation-time"]
+  transform: >
+    $.format = "str";
+```
+
+### Change new SMB file parameters to use default values
+TODO: Verify these default values are correct
+``` yaml
+directive:
+- from: swagger-document
+  where: $.parameters.FileCreationTime
+  transform: >
+    $.format = "str";
+    $.default = "now";
+- from: swagger-document
+  where: $.parameters.FileLastWriteTime
+  transform: >
+    $.format = "str";
+    $.default = "now";
+- from: swagger-document
+  where: $.parameters.FileAttributes
+  transform: >
+    $.default = "none";
+- from: swagger-document
+  where: $.parameters.FilePermission
+  transform: >
+    $.default = "inherit";
+```
+
+### FileRangeWriteFromUrl Constant
+``` yaml
+directive:
+- from: swagger-document
+  where: $.parameters.FileRangeWriteFromUrl
+  transform: >
+    delete $.default;
+    delete $["x-ms-enum"];
+    $["x-ms-parameter-location"] = "method";
```

Comment:
please add a comment explaining this change

Reply:
This value is supposed to be the constant value `update` and these changes turn it from a parameter into a constant.  We'll update with a comment.